pom.xml

```diff
@@ -246,6 +247,33 @@
           <skip>${format.skip}</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <version>${version.docker.plugin}</version>
+        <configuration>
+          <images>
+            <image>
+              <name>opensearch-custom-plugin</name>
+              <build>
+                <dockerFile>${project.basedir}/Dockerfile</dockerFile>
+                <tags>
+                  <tag>2.11.0</tag>
+                </tags>
+              </build>
+            </image>
+          </images>
+        </configuration>
+        <executions>
+          <execution>
+            <id>build</id>
+            <phase>initialize</phase>
```

Comment:
this should "pre-build" the image for us and then it is referenced in the application.properties so that we can use the plugins in analyzer configs...